plugins/google-sheets/src/pages/MapSheetFields.tsx

```diff
@@ -52,13 +52,13 @@ const inferFieldType = (cellValue: CellValue): CollectionFieldType => {
     if (typeof cellValue === "number") return "number"
 
     if (typeof cellValue === "string") {
-        const cellValueLowered = cellValue.toLowerCase()
+        const cellValueLowered = cellValue.trim().toLowerCase()
 
+        // If the cell value contains a newline, it's probably a formatted text field
+        if (/\n/.test(cellValueLowered)) return "formattedText"
```

Comment:
Any reason not to combine with the other regex? For sake of an early return?

Reply:
> For sake of an early return?

Yes, that's correct. It's a small optimization win!

Reply:
Makes sense! Speaking of, maybe this?

```suggestion
        // If the cell value contains a newline, it's probably a formatted text field
        if (cellValueLowered.includes("\n")) return "formattedText"
```

Reply:
Nice one!